core/domain/classifier/LDAStringClassifierTest.py

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from core.domain.classifier.lda_string_classifier import LDAStringClassifier
+from core.domain.classifier.LDAStringClassifier import LDAStringClassifier
```

Comment:
Oh, thanks for spotting that. Good that we found this after having missed it in Yang Lu's PR.